### ISSUE TITLE
NOT WORK!!!

### ISSUE THREAD

**Issue by Davidm16** [zhuowei/MCPELauncher#453]
Entity.setCape() not works
Entity.setRenderType() not works
Player.getExp() not works

**Comment by officialmcpro32** [comment]
*does not

**Comment by Davidm16** [comment]
Sí, soy español, a ver si tu eres capaz de hablar mi idioma. ¬¬

**Comment by zhuowei** [comment]
setCape is no longer supported and will never work again
setRenderType: do you have an example script?
getExp: do you have an example script?

**Comment by zhuowei** [comment]
Player.getExp() definitely works: I tested it; it returned 0.45 and the exp bar was 45% full, so it's working

**Comment by officialmcpro32** [comment]
@zhuowei I used Level.destroyBlock(x,y,z,false); on destroy why is it still dropping itself also why you removed setCape? @zhuowei

**Comment by Davidm16** [comment]
@zhouwei  I want to get Exp per 1000 like Player.setExp(2500), no percent.

Renders switches. Example: EntityRenderType.minecart  switches to  EntityRenderType.boat  and more.

So another renders like slime or spider don't appear, is buggy.

**Comment by zhuowei** [comment]
@officialmcpro32 that's odd - are you sure you're on the latest beta?

**Comment by officialmcpro32** [comment]
@zhuowei Yes it always drop itself especially when a pickaxe is used

**Comment by zhuowei** [comment]
@Davidm16 setExp also takes a percentage; only addExp takes the number of exp points. What are you trying to do?

**Comment by Davidm16** [comment]
I'm making an expchest. So I want to get exp points ‰ from the bar.
